Car_owners/tests/test_views.py

```diff
@@ -0,0 +1,269 @@
+import datetime
+import pytest
+from rest_framework import status
+from rest_framework.test import APIClient
+from application.views import OwnerViewSet, CarViewSet
+
+
+@pytest.fixture
+def api_client() -> APIClient:
+    yield APIClient()
+
+
+@pytest.fixture
+def owner_data() -> dict[str, str]:
+    owner_data = {"name": "Andrzej", "surname": "Starczyk", "phone": "575849675"}
+    return owner_data
+
+
+@pytest.fixture
+def car_data(api_client: APIClient, owner_data: dict[str, str]) -> dict[str, str | int]:
+    response_create_owner = api_client.post(
+        "/app/owners/", data=owner_data, format="json"
+    )
+    owner_id = response_create_owner.data["id"]
+    car_data = {
+        "brand": "Ford",
+        "model": "Mondeo",
+        "production_date": "2023-01-01",
+        "owner": owner_id,
+    }
+
+    return car_data
+
+
+@pytest.mark.django_db
+def test_create_and_get_owner(
```

Comment:
I would suggest dividing that into 2 tests (SOLID)

Reply:
The idea was to check, for example, if 'post' functionality works fine and to do that, inside the same test, I tried to 'get' the particular owner after 'post' by id to make sure, that 'post' really created an owner. But ok, I get your point.